pkg/util/util.go

```diff
@@ -1424,6 +1424,13 @@ func GetEnvVariable(env *[]corev1.EnvVar, name string) *corev1.EnvVar {
 	return nil
 }
 
+func GetAnnotationValue(annotations map[string]string, name string) string {
```

Comment:
It is worth returning a boolean `exists` as a second return value, the same as the map return value.

Reply:
see this change in Kaustav's PR
https://github.com/noobaa/noobaa-operator/pull/1389/files#diff-0fa449830b5dc2b274b7c95a58b5071edc23f1e44af84a99c1fa9e600767829fR2196

Reply:
ack, changed.